scripts/pipeline/orthology_benchmark.py

```diff
@@ -157,12 +160,49 @@ def get_core_names(species_names: List[str], host: str, port: int, user: str) ->
     return core_names
 
 
-def prep_input_for_orth_tools():
-    """Docstring"""
+def prep_input_for_orth_tools(source_dir: str, target_dir_ofinder: str) -> None:
+    """Prepares input files for selected orthology inference tools.
 
+    For OrthoFinder: Creates symlinks to input fasta files in `target_dir_ofinder`.
+    # For OMA: ...
 
-def run_orthology_tools():
-    """Docstring"""
+    Args:
+        source_dir: Path to the directory containing fasta files.
+        target_dir_ofinder: Path to the directory where symlinks to fasta files will be created
+            for OrthoFinder to use.
+
+    Raises:
+        FileExistsError: If directory `target_dir_ofinder` already exists.
+        FileNotFoundError: If directory `source_dir` does not exist or does not contain any fasta files.
+        subprocess.CalledProcessError: If creating symlinks fails for some other reason.
+    """
+    if not os.path.isdir(source_dir):
+        raise FileNotFoundError("Directory containing fasta files not found.")
+    if len(fnmatch.filter(os.listdir(source_dir), '*.fasta')) == 0:
+        raise FileNotFoundError("No fasta files found.")
+
+    # OrthoFinder
+    os.mkdir(target_dir_ofinder)
+    script_symlinks = os.path.join(Path(__file__).parents[2], "scripts", "pipeline", "symlink_fasta.py")
+    subprocess.run([script_symlinks, "-s", target_dir_ofinder, "-d", source_dir],
+                   capture_output=True, check=True)
+
+
+def run_orthology_tools(input_ofinder: str) -> None:
+    """Runs selected orthology inference tools: OrthoFinder.
+
+    Args:
+        input_ofinder: Path to the directory containing symlinks to the input fasta files for OrthoFinder.
```

Comment:
```suggestion
        input_ofinder: Path to the directory containing the input fasta files for OrthoFinder.
```
Whilst in your script and methods the input is prepared using symlinks, and this is recommended based on our useage - it does not have to be the case, and running this function should have no impact on the content of the directory for OrthoFinder, so we can be more generic here.

Reply:
```suggestion
        input_dir: Path to the directory containing the input .fasta files for orthology tools.
```